### PR TITLE
classic_setter pattern and required files

### PR DESCRIPTION
**Pull request by silverCase**
Added new classic_setter pattern, tests, __init__.py files for other patterns

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#112](https://github.com/yegor256/aibolit/issues/112) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/7c535e1a-f09a-440f-88b0-7a6dd6b6d402); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @acheshkov[/z](https://www.0crat.com/u/acheshkov) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: test/patterns/classic_setter/test_classic_setter.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from pathlib import Path
+
+
+class TestSetter(TestCase):
+    dir_path = Path(os.path.realpath(__file__)).parent
+    setter_finder = ClassicSetter()
+
+    def test_one_valid_patterns(self):
+        lines = self.setter_finder.value(Path(self.dir_path, 'BaseKeyframeAnimation.java'))
+        assert lines == [40]
```
Comment (tribals) [inline test/patterns/classic_setter/test_classic_setter.py]: Please, stop using "bare" `assert` statement, and use `unittest.TestCase` [assertion methods](https://docs.python.org/3/library/unittest.html#unittest.TestCase) instead.

File: test/patterns/classic_setter/test_classic_setter.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    def test_four_setter_patterns(self):
+        lines = self.setter_finder.value(Path(self.dir_path, 'Configuration.java'))
+        assert lines == [1236, 1240, 3783, 3819]
```
Comment (tribals) [inline test/patterns/classic_setter/test_classic_setter.py]: Same

File: test/patterns/classic_setter/test_classic_setter.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    def test_another_setter_patterns(self):
+        lines = self.setter_finder.value(Path(self.dir_path, 'SequenceFile.java'))
+        assert lines == [2849, 2855, 2861, 3127]
```
Comment (tribals) [inline test/patterns/classic_setter/test_classic_setter.py]: Same

**Comment by tribals** [comment]
@acheshkov Please, pay attention to my comments

**Comment by acheshkov** [comment]
@silverCase Oleg, please refactor test cases. Use 'assertEqual' instead of 'assert'.

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 3 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: performer @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)[/z](https://www.0crat.com/u/fanifieiev) didn't make CR comments

**Pull request merged**